src/mumble/main.cpp

```diff
@@ -713,6 +713,8 @@ int main(int argc, char **argv) {
 		AudioWizard *aw = new AudioWizard(Global::get().mw);
 		aw->exec();
 		delete aw;
```

Comment:
On an unrelated note, we should use a smart pointer here.

Reply:
Done